lib/atom-inline-blame.js

```diff
@@ -1,9 +1,11 @@
 'use babel';
 
-import InlineBlameView from './InlineBlameView';
+import { CompositeDisposable } from "atom";
+
+let InlineBlameView;
 
 export default {
-  watchedEditors: null,
+  watchedEditors: [],
```

Comment:
Use a ```Set```.

Reply:
Any specific reason? `Array`s are faster, and we're already enforcing uniqueness on line 27

Reply:
`Set`s are faster than `Array`s when performing a search. O(1) vs O(n). But given the size of the array it does not really matter that much. 😆